lib/pbench/server/auth/__init__.py

```diff
@@ -280,23 +301,23 @@ def token_introspect_offline(
 
     def get_userinfo(self, token: str = None) -> JSON:
         """
-        The userinfo endpoint returns standard claims about the authenticated user,
-        and is protected by a bearer token.
+        The userinfo endpoint returns standard claims about the authenticated
+        user, and is protected by a bearer token.
         http://openid.net/specs/openid-connect-core-1_0.html#UserInfo
         Args:
             token: Valid token to extract the userinfo
         Returns:
             Userinfo payload
             {
-                "family_name": "family_name",
+                "family_name": <surname>,
```

Comment:
OK, I like the angle-brace placeholders you're using here; but most of the other example return payloads still use sample text, including in a few places comments  (e.g., `# seconds`) which wouldn't be necessary if you used `"<seconds>"` or `<seconds-as-string>` as the value. (Plus it'd be more clear where the return is a OIDC constant, like presumably `"openid email profile"` or `"offline"` vs a sample value like `"user@example.com"`.